app-arch/zstd/patches/zstd-1.5.5.patchset

```diff
@@ -0,0 +1,44 @@
+From 85212c5fc1df444995b2c599a2b68931c751b482 Mon Sep 17 00:00:00 2001
+From: Augustin Cavalier <waddlesplash@gmail.com>
+Date: Wed, 29 Sep 2021 17:58:52 -0400
+Subject: Disable weak symbols; they do not seem to behave correctly.
```

Comment:
I think this patch can be dropped after beta4.

Reply:
Dropped, thanks for commenting!